hs_tools_resource/models.py

```diff
@@ -35,190 +33,112 @@ def metadata(self):
         md = ToolMetaData()
         return self._get_metadata(md)
 
-
 processor_for(ToolResource)(resource_processor)
 
 
 class RequestUrlBase(AbstractMetaDataElement):
-    term = 'Request Url Base'
-    value = models.CharField(null=True, max_length="500") # whatever the user gives us- format is "http://www.example.com/{resource-info}"
-
+    term = 'RequestUrlBase'
+    value = models.CharField(max_length=1024, null=True)
+    resShortID = models.CharField(max_length=128, default="UNKNOWN")
```

Comment:
@shawncrawley If the web app resource is a HydroShare resource, then we can use a foreign key relationship, no?